src/app/src/components/Visualizations.jsx

```diff
@@ -54,6 +54,21 @@ const Visualizations = () => {
         }
     }, [showSurvey, surveyHasBeenDisplayed, dispatch]);
 
+    const config = CONFIG[geoMode];
```

Comment:
Will this crash if `geoMode` is `null` or `undefined`?

Reply:
In a number of locations the app will crash if `geoMode` is `null` or `undefined`. It is set when the site loads and should never be `null`, but I'm going to add a follow-up card to generally add some protections around this.